core/app/models/spree/tax/order_adjuster.rb

```diff
@@ -0,0 +1,21 @@
+module Spree
+  module Tax
+    class OrderAdjuster
+      attr_reader :order
+
+      include TaxHelpers
+
+      def initialize(order)
+        @order = order
+      end
+
+      def adjust!
+        return unless order_tax_zone
+
+        (order.line_items + order.shipments).each do |item|
```

Comment:
We could, potentially. I shied away of adding more methods to `Spree::Order`. I'd do that on a later commit though once we actually see it DRYing away other things, namely when I have to write a method that does the same thing for the shipping rates in a shipment (https://github.com/solidusio/solidus/pull/759).